### PR TITLE
m: Fix various clippy warnings

### PR DESCRIPTION
**Pull request by notgull**
Fixes a handful of Clippy warnings that showed up when I ran it on this codebase.

### CODE REVIEW

File: examples/find-system-font.rs [hunk 1]
```diff
@@ -38,7 +38,7 @@ fn main() {
             }
         }
         None => {
-            println!("Error: '{}' not found.", FAMILY_NAME);
+            println!("Error: '{FAMILY_NAME}' not found.");
```
Comment (RazrFalcon) [inline examples/find-system-font.rs:41]: This is way too modern syntax. I think clippy already reverted this check.

**Comment by RazrFalcon** [comment]
This commit should be update for merge.

**Comment by notgull** [comment]
> This commit should be update for merge.

Done!

**Comment by RazrFalcon** [comment]
Thanks.

**Pull request merged**